src/RootViewModel.js

```diff
@@ -58,6 +59,15 @@ export class RootViewModel extends ViewModel {
         } else {
             const oldLink = this.link;
             this.link = Link.parse(hash);
+            const matrixUrl = this.link.toMatrixUrl()
+            if (matrixUrl) {
+                this.hidden = true;
+                setTimeout(() => {
```

Comment:
perhaps pass `setTimeout` in as a view model option in case we want to unit test these one day